src/targets/browser/index.jsx

```diff
@@ -57,17 +55,12 @@ const setupAppContext = memoize(() => {
   })
   client.registerPlugin(RealtimePlugin)
   client.registerPlugin(flag.plugin)
-  // We still need to init cozy-client-js for the Uploader
-  cozy.client.init({
-    cozyURL: cozyUrl,
-    token: data.token
-  })
 
   let middlewares = [thunkMiddleware, loggerMiddleware]
 
   // Enable Redux dev tools
   const composeEnhancers =
-    (__DEVELOPMENT__ && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) || compose
+    (flag('debug') && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) || compose
```

Comment:
As the flag plugin is initialized on client's `login` event and because `client.login()` is asynchronous, it seems like relying on `flag('debug')` here would be subject to race condition, wouldn't it?

I suppose it would work here because on web apps, the `client.login()` only access local resources, so it would be fast enough to ends before the composeEnhancers initialisation. But it seems a bit risky.